src/dcat-us/constants/contexts.ts

```diff
@@ -0,0 +1,59 @@
+// Context header for DCAT US 1.1
+export const HEADER_V_1X = {
+    '@context':
+        'https://project-open-data.cio.gov/v1.1/schema/catalog.jsonld',
+    '@type': 'dcat:Catalog',
+    conformsTo: 'https://project-open-data.cio.gov/v1.1/schema',
+    describedBy: 'https://project-open-data.cio.gov/v1.1/schema/catalog.json',
+};
+
+// Context header for DCAT US 3.0
+// source: https://raw.githubusercontent.com/DOI-DO/dcat-us/refs/heads/main/context/dcat-us-3.0.jsonld
+export const HEADER_V_3X = {
+    '@context': {
+        '@version': 1.1,
```

Comment:
Should the version be `3.0`?

Reply:
@sansth1010 good call on the header const, though I was actually talking about line 14 😅
`@version: 1.1` -> `@version: 3.0`

Reply:
It seems the [source](https://raw.githubusercontent.com/DOI-DO/dcat-us/refs/heads/main/context/dcat-us-3.0.jsonld) uses `@version: 1.1` . I copied required context fields from the source.

Reply:
Oh dang... well either that's a crazy coincidence or the DCAT spec team did some inadvertent copy-pasta 😂

Thanks for the clarification, ignore this comment.